src/toolbox/charts.js

```diff
@@ -179,13 +193,12 @@ const remixerForChartType = {
    * DONUT CHART
    * 
    * Requirements:
-   * - measure cannot have SUM or UNKNOWN aggregation type
+   * - measure cannot have UNKNOWN, AVERAGE, MEDIAN, or NONE aggregation type
    */
   donut(dg, chartLimits) {
     const allowedMeasures = dg.measureSets.filter(
-      measureSet => !includes(["SUM", "UNKNOWN"], measureSet.measure.aggregatorType)
+      measureSet => !includes(["UNKNOWN", "AVG", "AVERAGE", "MEDIAN", "NONE"], measureSet.measure.aggregatorType)
```

Comment:
I'd leave it to check for SUM only instead, because there are other aggregation types available but not frequently used, and soon calculations will also have their own type.